src/daemon-base/__CEPH_BASE_PACKAGES__

```diff
@@ -1,14 +1,8 @@
 \
-        ca-certificates \
-        e2fsprogs \
         ceph-common__ENV_[CEPH_POINT_RELEASE]__  \
         ceph-mon__ENV_[CEPH_POINT_RELEASE]__  \
         ceph-osd__ENV_[CEPH_POINT_RELEASE]__ \
         ceph-mds__ENV_[CEPH_POINT_RELEASE]__ \
         rbd-mirror__ENV_[CEPH_POINT_RELEASE]__  \
-        __CEPH_MGR_PACKAGE__\
-        kmod \
```

Comment:
modprobe & modinfo can be called by ceph

Reply:
I'm pretty sure that when I checked, `kmod` wasn't a dependency for installing ceph binaries. This to me would seem to be a bug in Ceph that it doesn't have these as dependencies if they are needed rather than a bug in ceph-container.

Reply:
@ktdreyer What's your though on ceph.spec which doesn't requires kmod while the source code does some modprobe or modinfo ?

Reply:
Where does it do that?